tests/console/yast2_ftp.pm

```diff
@@ -107,20 +163,37 @@ sub run {
 
     assert_screen 'yast2_ftp_create_upload_dir_confirm';    # confirm to create upload directory
     wait_screen_change { send_key 'alt-y' };
+    assert_screen 'yast2_ftp_expert_settings';              # check passive mode value and enable SSL
     wait_screen_change { send_key 'alt-m' };
-    type_string_slow "30000\n";
+    type_string_slow($vsftpd_directives->{pasv_min_port} . "\n");
     wait_screen_change { send_key 'alt-a' };
-    type_string_slow "30100\n";
-    wait_screen_change { send_key 'alt-l' };                # enable SSL
-    assert_screen 'yast2_ftp_expert_settings';              # check passive mode value and enable SSL
+    type_string_slow($vsftpd_directives->{pasv_max_port} . "\n");
+    wait_screen_change { send_key('alt-l', 3) };            # enable SSL, and wait with next step
     wait_screen_change { send_key 'alt-s' };                # give path for DSA certificate
-    type_string '/etc/vsftpd.pem';
-    wait_screen_change { send_key 'alt-p' };                # open port in firewall
+    type_string_slow($vsftpd_directives->{rsa_cert_file});
+    assert_screen 'yast2_ftp_port_closed';
+    send_key 'alt-p';                                       # open port in firewall
+    assert_screen 'yast2_ftp_port_opened';
     send_key 'alt-f';                                       # done and close the configuration page now
 
     # yast might take a while on sle12 due to suseconfig
     die "'yast2 ftp-server' didn't exit with zero exit code in defined timeout" unless wait_serial("yast2-ftp-server-status-0", 180);
 
+    # open passive ftp ports manually
+    if (is_sle('15+')) {
+        record_soft_failure('bsc#1084434');
+        # use pre-defined /usr/lib/firewalld/services/ftp.xml to enable firewalld ftp service & reload firewalld
+        assert_script_run('firewall-cmd --permanent --service=vsftpd --add-port=30000-30100/tcp');
+        #assert_script_run('firewall-cmd --permanent --add-service=vsftpd');
```

Comment:
don't add disabled code

Reply:
I have missed that. Thanks! Should be fixed by now